app/services/claim_service.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+require_dependency 'evss/claims_service'
+
+class ClaimService
+  EVSS_CLAIM_KEYS = %w(openClaims historicalClaims).freeze
+
+  def initialize(user)
+    @user = user
+  end
+
+  def all
+    raw_claims = client.all_claims.body
+    EVSS_CLAIM_KEYS.each_with_object([]) do |key, claims|
+      next unless raw_claims[key]
+      claims << raw_claims[key].map do |raw_claim|
+        find_or_initialize_claim(raw_claim)
+      end
+    end.flatten
+  rescue Faraday::Error::TimeoutError, Timeout::Error
+    Claim.where("data->>'participant_id' = ?", @user.participant_id)
+  end
+
+  def find_by_evss_id(id)
+    raw_claim = client.find_claim_by_id(id).body.fetch('claim', {})
+    find_or_initialize_claim(raw_claim)
+  rescue Faraday::Error::TimeoutError, Timeout::Error
```

Comment:
I'm not sure which errors we want to rescue from and which ones we want to propagate and am still not 100% sure on all the different errors we're receiving from EVSS (are they fatal/should alert user or just downtime related?). I'm also assuming we want to log something here even though we aren't re-raising so that excessive timeouts don't go unnoticed
@saneshark any input?